ebu_tt_live/scripts/ebu_run.py

```diff
@@ -0,0 +1,47 @@
+"""
+The ``ebu-run`` script is a universal runner for any component or complex interconnected sets of components
+based on a compliant configuration file or set of valid options. The configuration framework uses
+the mozilla/configman package that picks up configuration from command line/config file.
+
+Basic usage:
+------------
+
+  ::
+
+    ebu-run --admin.conf=ebu_tt_live/examples/config/simple-producer.json
+
```

Comment:
This is just a line too run it with a config file. Obviously there are multiple example configs in the very same folder